Wizardry/Core/BattleSys/Source/BattleHit.c

```diff
@@ -450,6 +450,15 @@ STATIC_DECLAR int BattleHit_CalcDamage(struct BattleUnit * attacker, struct Batt
     }
 #endif
 
+#if defined(SID_Expertise) && (COMMON_SKILL_VALID(SID_Expertise))
+    if (BattleSkillTester(defender, SID_Expertise))
+    {
+        if(gBattleHitIterator->attributes & BATTLE_HIT_ATTR_CRIT)
```

Comment:
code style, use `crit_atk` is recommended

Reply:
Done